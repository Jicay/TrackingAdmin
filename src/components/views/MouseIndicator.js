import React, {Component} from "react";

export default class MouseIndicator extends Component {

    constructor(props) {
        super(props);
    }

    render() {
        const {top, left} = this.props;
        const style = {
            position: 'relative',
            left: left + 'px',
            top: top + 'px',
            zIndex: 10
        }

        return (<div className="mouse-indicator" style={style}></div>)
    }

}
