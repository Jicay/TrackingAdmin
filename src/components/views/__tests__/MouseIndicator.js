import React from 'react';
import renderer from 'react-test-renderer';
import MouseIndicator from "../MouseIndicator";

it('renders correctly', () => {
    const tree = renderer
        .create(<MouseIndicator top={10} left={87}/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});