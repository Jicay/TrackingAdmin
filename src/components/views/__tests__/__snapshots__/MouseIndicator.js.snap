// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mouse-indicator"
  style={
    Object {
      "left": "87px",
      "position": "relative",
      "top": "10px",
      "zIndex": 10,
    }
  }
/>
`;
